src/app/Services/Tenant.php

```diff
@@ -27,18 +27,18 @@ public static function get()
         return Company::find(self::tenantId());
     }
 
+    public static function tenantDatabase()
```

Comment:
this is not intended to be public. Should also be below `tenantPrefix()`

Reply:
the public method is used within the `ConnectionStoragePath` trait so it cannot be made private without further refactor.